crates/sargon/src/profile/v100/factors/factor_source_kind.rs

```diff
@@ -118,6 +118,20 @@ impl FactorSourceKind {
     }
 }
 
+impl FactorSourceKind {
+    pub fn display_order(&self) -> u8 {
```

Comment:
IMO in the context of this PR, it should be specific display order for Select Factor for Access Role, not generic order.